recipes/karateclub/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "karateclub" %}
+{% set version = "1.0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1278228f620e3e1dd6271c34912055223661146d81373b0de74fb661166fbf49
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pytest-runner
+    - gensim
+    - networkx
+    - numpy
+    - pandas
+    - pip
+    - pygsp
+    - python
+    - python-louvain
+    - scikit-learn
+    - scipy
+    - six
+    - tqdm
+  run:
+    - gensim
+    - networkx
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - pygsp
+    - python
+    - python-louvain
+    - scikit-learn
+    - scipy
+    - six
+    - tqdm
+
+test:
+  imports:
+    - karateclub
+    - karateclub.community_detection
+    - karateclub.community_detection.non_overlapping
+    - karateclub.community_detection.overlapping
+    - karateclub.dataset
+    - karateclub.graph_embedding
+    - karateclub.node_embedding
+    - karateclub.node_embedding.attributed
+    - karateclub.node_embedding.meta
+    - karateclub.node_embedding.neighbourhood
+    - karateclub.node_embedding.structural
+    - karateclub.utils
+  requires:
+    - mock
+    - pytest
+    - pytest-cov
+
+about:
+  home: "https://github.com/benedekrozemberczki/karateclub"
+  license: GPL-3.0-only
+  license_file: LICENSE.txt
```

Comment:
Include a comment above the `license_file` line containing the following link as a reminder to check if the license file is included in a future release of the source distribution:

https://github.com/benedekrozemberczki/karateclub/pull/39

Something like:

```
   # Check status of https://github.com/benedekrozemberczki/karateclub/pull/39 for inclusion of license file in source distribtion
   license_file: LICENSE.txt
   ...
```